README.md

```diff
@@ -27,7 +27,7 @@ To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 To get a local copy, you should have installed a code editor like [Visual Studio Code](https://code.visualstudio.com/download), [Node.js](https://nodejs.org/en/download/) and [Git](https://git-scm.com/downloads).
-### Setup
+### Clone this repository
 - Clone the repository using: ``` git clone git@github.com:OscarFMdev/mathMagicians.git ```
```

Comment:
- [ ] OPTIONAL
 Kindly include instructions in your ReadMe on how to run the test cases of your project, under the _Getting Started_ section.

Reply:
@rloterh I erase those instructions because previous code reviewers told me to erase them 😅 But I'll add them again, thank you!